function-reference/build-functions/text.xml

```diff
@@ -24,7 +24,27 @@ during the unpack and compile stages.
       <c>unpack archives</c>
     </ti>
     <ti>
-      Unpack the specified archives.
+      <p>
+        Unpack the specified archives. Recognised formats are:
+      </p>
+      <ul>
+        <li><c>*.tar</c></li>
+        <li>
+          <c>*.gz</c>, <c>*.Z</c>,
+          <c>*.tar.gz</c>, <c>*.tgz</c>, <c>*.tar.Z</c>
+        </li>
+        <li>
+          <c>*.bz2</c>, <c>*.bz</c>,
+          <c>*.tar.bz2</c>, <c>*.tbz2</c>, <c>*.tar.bz</c>, <c>*.tbz</c>
+        </li>
+        <li><c>*.lzma</c>, <c>*.tar.lzma</c></li>
+        <li><c>*.xz</c>, <c>*.tar.xz</c>, <c>*.txz</c></li>
+        <li><c>*.zip</c>, <c>*.jar</c></li>
+        <li><c>*.7z</c></li>
```

Comment:
Maybe it'd be useful to mention for which formats you need extra deps.

Reply:
Not sure. It is the "function reference" section, so I suppose that explanations should be brief. Maybe move the whole list to `ebuild-writing/functions/src_unpack`, with a longer explanation?

Reply:
Makes sense.

Reply:
I've moved it to the other section, but have only added an `<important>` box. We can add more information later, but this is not the goal of this PR and shouldn't block it.